modules/nf-core/pilon/tests/main.nf.test

```diff
@@ -0,0 +1,38 @@
+nextflow_process {
+
+    name "Test Process PILON"
+    script "../main.nf"
+    process "PILON"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pilon"
+
+
+    test("Test PILON") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:false ], // meta map
+                            file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
```

Comment:
Apparently the new guideline is to use URLs instead of the test_data object, so please change the tests accordingly

Reply:
Hi, I just started so can you point to the guidelines? I found some info (but not technical details) [here](https://nf-co.re/docs/contributing/test_data_guidelines)?

Reply:
Changing to the test-data path is not required currently, just optional.